core/src/test/java/fr/sncf/osrd/utils/graph/AStarTests.kt

```diff
@@ -24,16 +23,68 @@ class AStarTests {
     @Test
     @Throws(Exception::class)
     fun lessBlocksVisitedWithHeuristic() {
-        val infra = fullInfraFromRJS(getExampleInfra("small_infra/infra.json"))
-        val origin = mutableSetOf(convertRouteLocation(infra, "rt.DA2->DA5", Offset(0.meters)))
-        val destination =
-            mutableSetOf(convertRouteLocation(infra, "rt.DH2->buffer_stop.7", Offset(0.meters)))
+
+        // ---------- DUMMY INFRA ----------
+
+        val dummyInfra = DummyInfra()
+        val fullDummyInfra = dummyInfra.fullInfra()
+
+        val pointsList =
+            hashMapOf(
+                "a" to Point(.0, .0),
```

Comment:
This doesn't fix the problem of the previous test, which is that the geographic coordinates don't match the block lengths. The default length is still used (100m) but the geo distances would be thousands of kilometers there.

To make that work, we need to compute the actual geo distance between each point when instantiating each blocks.

Reply:
added the distances